components/calculation/MeasureHighlightingPanel.tsx

```diff
@@ -32,28 +27,36 @@ export default function MeasureHighlightingPanel({ patientId }: MeasureHighlight
   const { classes } = useStyles();
   const [searchValue, setSearchValue] = useState('');
   const detailedResultLookup = useRecoilValue(detailedResultLookupState);
-  const { parsedHTML, defIds } = useMemo(() => {
-    const defIds: Record<string, string> = {};
+  const parsedHTML = useMemo(() => {
     const parsedHTML = parse(detailedResultLookup[patientId]?.detailedResults?.[0].html || '', {
       replace: elem => {
-        if (elem.type === 'text') {
-          const data = (elem as DomText).data;
-          if (data.match(expressionDefRegex)) {
-            const splitData = data.split('"');
-            defIds[splitData[1]] = splitData[1].toLowerCase().replace(' ', '-');
-            return <span id={defIds[splitData[1]]}>{data}</span>;
-          }
+        if ((elem as DomElement).attribs?.['data-statement-name']) {
+          const statementName = (elem as DomElement).attribs?.['data-statement-name'];
+          const libraryName = (elem as DomElement).attribs?.['data-library-name'];
```

Comment:
Since we do a check in the `if` statement that `attribs` is defined, could we change these to `(elem as DomElement).attribs['data-statement-name']` and `(elem as DomElement).attribs['data-library-name']`?

Reply:
Nice catch!